bin/addons/base/migrations/6.0.1.3/post-migration.py

```diff
@@ -302,7 +302,7 @@ def migrate(cr, version):
     pool = pooler.get_pool(cr.dbname)
     set_defaults_on_act_window(cr)
     set_defaults_on_ir_attachment(cr)
-    openupgrade.set_defaults(cr, pool, defaults)
+    openupgrade.set_defaults(cr, pool, defaults,False,True)
```

Comment:
Without the use_orm to True the process fails in with company_ids in res_user with:

"UPDATE res_users SET company_ids = ARRAY[1]"

Probably because the defaults function is not exactly the same it was when openupgrade class was not in openupgradelib. So it is related to the move from openupgrade class to opeenupgradelib.openupgrade.